scripts/update_changelog.sh

```diff
@@ -178,6 +179,8 @@ if [ $? -eq 0 ]; then
 
             fields=$(echo "${update_details["fields"]}" | jq -r 'to_entries[] | "\(.key): \(.value)"')
 
+            get_user_full_name "$space_id" "${update_details["user_id"]}" "$CONTENTFUL_TOKEN"
+
             {
             echo "# CMS Update: $formatted_timestamp"
             echo ""
```

Comment:
I think content type can be taken from the update_details["content_type"] field. Now you are using the CMS API that value will be available. At the moment you are printing as N/A.

Also I think revision could be retrieved from sys.version when the changeType is deletedEntry so in your new function get_deleted_entry_by_id you can do:
update_details["revision"]=$(echo "$cleaned_response" | jq -r '.sys.version')
and then use that instead of N/A in the output.

Reply:
Also could you add the following comment at the end of the block comment at the top of the file to guide on the version of bash required please?

# This script requires bash 4.x or greater to run as it uses associative arrays.
# The default shell on a mac is zsh so you should install a later version of bash using brew install bash 
# and run the script using: 
# /usr/local/bin/bash ./scripts/update_changelog.sh preprod